strr-host-pm-web/app/stores/propertyRequirements.ts

```diff
@@ -78,7 +78,7 @@ export const usePropertyReqStore = defineStore('property/requirements', () => {
       propertyReqs.value = res
 
       // TODO: confirm these are the only scenarios we want to open the form without further user interaction
-      if (!res.isStrProhibited || !!res.isStraaExempt) {
+      if (!res.isStrProhibited && !res.isStraaExempt) {
```

Comment:
My understanding of these rules is a little hazy, can you confirm that this works as expected with the 'Add supporting information' tab and 'review / confirm' tab? Both of those use showUnitDetailsForm when deciding whether or not to show 'no documents required' vs 'incomplete'

Reply:
Sure thing